weatherman.py

```diff
@@ -0,0 +1,158 @@
+import os
+import csv
+import argparse
+from colorama import Fore, Style
+
+def convert_month(month_input):
+    month_names = [
+        "Jan", "Feb", "Mar", "Apr", "May", "Jun",
+        "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
+    ]
+    if month_input.isdigit() and 1 <= int(month_input) <= 12:
+        return month_names[int(month_input) - 1]
+    return None
```

Comment:
we are returning None here because if user enters any value greater than 12 that it will return None and print no valid data found for the given month